lib/Linter.js

```diff
@@ -612,7 +652,7 @@ module.exports = (function() {
      *      or null if not.
      * @private
      */
-    function parse(text, config, filePath) {
+    parse(text, config, filePath) {
```

Comment:
I am in agreement with private stuff to have `_`. 
You know what, i am just going to move this function outside the class as a private function as its not part of the API and we dont want to expose unnecessary.